admin/cypress/integration/kuba_admin_spec.js

```diff
@@ -6,43 +6,43 @@ describe("Kuba Admin Test", () => {
 
   it("Post user with bad email and no password", () => {
```

Comment:
`Describe`s can be imbricated, like : 
```js
describe("Kuba Admin Test", () => {
    describe("Post user", () => {
        it("with bad email", () => {...});
        it("with bad password", () => {...});
    })
})
```
It's easier to read on test report and help to organize your tests.

Reply:
Also, /Matthieu mode on/ the name of the test could be a little more explicit, like "should display error message when email is invalid". And I think it's better to use the word 'Create' instead of 'Post' (like the name of the button) /Matthieu mode off/